### PR TITLE
Coe104 testing

### PR DESCRIPTION
**Pull request by cessien66**
Testing requirement for file(type) upload without an existing csv(yet)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@testing-library/jest-dom": "^4.2.0",
```
Comment (braunsonm) [inline package.json]: Why were these added? they should already be included. Same with some of your jest stuff below

File: src/__tests__/FileConvention.test.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+import React, {ReactDOM} from 'react';
+import {mount, shallow} from 'enzyme';
+import ParserComponent from '../components/ParserComponent';
+import {FileType} from '../components/ParserInterface';
+
+describe('File type is uploaded correctly', () => {
+  describe('file input', () => {
+    const fileUploader = jest.fn();
```
Comment (braunsonm) [inline src/__tests__/FileConvention.test.tsx]: Unused?

File: src/__tests__/FileConvention.test.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    const comp = mount(
+        <ParserComponent
+          prompt={prompt}
+          fileType={fileType}
+        />
+    );
+    it('File csv', (fileType) => {
+      fileType(true);
+      expect({fileType});
```
Comment (braunsonm) [inline src/__tests__/FileConvention.test.tsx]: What does this test?

File: package.json (hunk 1 above)
Comment (cessien66) [reply package.json]: it was part of the dependencies I installed. Oversight

File: src/__tests__/FileConvention.test.tsx (hunk 3 above)
Comment (cessien66) [reply src/__tests__/FileConvention.test.tsx]: apologies. My saves were not saved. My laptop died. Working on it now
